### DISCUSSION TITLE
Amélioration de l'accessibilité des themes

### DISCUSSION THREAD

**Discussion by frouze** [JeromeDevome/GRR#388] [Idées]
Bonjour,
Je viens de constater que les couleurs des thèmes paramétrables depuis l'interface d'administration de GRR dans l'onglet 'configuration générale / apparence" (version GRR testée 4.3.5) ne respectent pas les critères de contraste liées à l'accessibilité numérique. Bien sûr il est possible de modifier la CSS après installation mais ça serait plus pertinent que ça soit conforme nativement. 
J'ai effectué les tests avec l'application 'Colour Contrast Analyser". Pouvez-vous y jeter un oeil svp ? 
Merci d'avance
frouze

**Comment by ynaessens** [comment]
Bonjour,
votre remarque relative à l'accessibilité visuelle s'applique-t-elle à tous les thèmes ?
Si aucun thème ne s'avère conforme, vous pouvez contribuer à l'amélioration de GRR en proposant une combinaison de couleurs qui satisfasse les critères d'accessibilité.
Cordialement,
YN
